src/main/java/com/minecolonies/coremod/commands/colonycommands/RefreshColonyCommand.java

```diff
@@ -7,6 +7,7 @@
 import com.mojang.authlib.GameProfile;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.mojang.authlib.GameProfile'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)